mkdocs/docs/development.md

```
@@ -40,9 +41,9 @@ Use `remote-containers.rebuildAndReopenInContainer` aka "Dev Containers: Rebuild
   sphinx-autobuild --host 0.0.0.0 --open-browser -b html "source" "build"
   ```
 
-  > `--host 0.0.0.0` is required in case if you running `sphinx-autobuild` in Dev Containers and accessing HTML using host browser. Docs will be served on http://127.0.0.1:8000/.
-  > There is also [`sphinx-serve`] Python module which also could be used for similar
-  > functionality.
+  > `--host 0.0.0.0` is required in case if you running `sphinx-autobuild` in Dev Containers and accessing HTML using
+  > host browser. Docs will be served on http://127.0.0.1:8000/. There is also [`sphinx-serve`] Python module which also
```

Markdownlint (0.35.0)

Bare URL used

(MD034, no-bare-urls)